mitosheet/mitosheet/code_chunks/export_to_file_code_chunk.py

```diff
@@ -21,8 +21,7 @@
 def get_format_code(state: State, sheet_indexes: Dict[int, str]) -> list:
     code = []
     formats = state.df_formats
-    for sheet_index in sheet_indexes.keys():
-        sheet_name = state.df_names[sheet_index]
+    for sheet_index, export_location in sheet_indexes.items():
```

Comment:
`sheet_indexes` is probably a badly named variable - everywhere else in the codebase, it is a list of sheet_indexes. 

Want to rename this to the name of the variable you are passing into this function?